samples/nrf9160/mqtt_simple/overlay-tls.conf

```diff
@@ -6,7 +6,8 @@
 
 # MQTT TLS
 CONFIG_MQTT_LIB_TLS=y
-CONFIG_MQTT_BROKER_PORT=8883
+CONFIG_MQTT_BROKER_PORT=8886
+CONFIG_MQTT_TLS_PEER_VERIFY=1
```

Comment:
```suggestion
```

Reply:
strangely this is no longer an issue. i must have tested incorrectly